plugins/woocommerce/src/Admin/Features/Features.php

```diff
@@ -146,6 +147,10 @@ public static function load_features() {
 				new $feature_class();
 			}
 		}
+
+		if ( FeaturesUtil::feature_is_enabled( 'blueprint' ) ) {
+			new \Automattic\WooCommerce\Admin\Features\Blueprint\Init();
+		}
```

Comment:
WCA feature flag system automatically initializes the feature class when the feature is enabled but core system doesn't do this so we need to manually initialize the feature class. I think we should initialize the feature class in the same place as WCA does so it's easier to maintain. Open to suggestions.

Reply:
Another option I see is to initialize it with the [DI container in the WooCommerce class](https://github.com/woocommerce/woocommerce/blob/ccc6eb86c76bafdb642f09c5f694e801dee9e414/plugins/woocommerce/includes/class-woocommerce.php#L314-L315). I'm good with what you have though.

Reply:
Thanks for the suggestion! 🙌   On second thought, I think it's better to keep it as is so we only initialize it in the admin context.